src/env-inl.h

```diff
@@ -613,6 +613,12 @@ inline void Environment::set_quic_state(std::unique_ptr<QuicState> buffer) {
   quic_state_ = std::move(buffer);
 }
 
+inline quic::QuicMonitor* Environment::quic_monitor() {
+  if (!quic_monitor_)
+    quic_monitor_.reset(new quic::QuicMonitor(this));
```

Comment:
Clang-tidy issue: 
`quic_monitor_ = std::make_unique<quic::QuicMonitor>(this);`